src/components/TalentPage/components/ListProofs/components/ProofBlock/ProofBlock.jsx

```diff
@@ -1,22 +1,22 @@
-import { useState } from "react";
 import s from "./ProofBlock.module.scss";
+import { Kudos } from "./components/Kudos";
 
 export function ProofBlock({ id, link, text, created }) {
     return (
         <div className={s.out}>
             <div className={s.proofs}>
                 <div className={s.info}>
                     <h1>Link:</h1>
-                    <a className={s.link} href={link} target="_blank" rel="noreferrer">
+                    <a className={s.link} href={link} target="_blank">
```

Comment:
return attribute rel="noreferrer"
```suggestion
                    <a className={s.link} href={link} target="_blank" rel="noreferrer">
```